docs/iris/src/whatsnew/3.0.2.rst

```diff
@@ -38,6 +38,10 @@ This document explains the changes made to Iris for this release
       developers to easily disable `cirrus-ci`_ tasks. See
       :ref:`skipping Cirrus-CI tasks`. (:pull:`4019`) [``pre-v3.1.0``]
 
+   #. `@pp-mo`_ adjusted the use of :func:`dask.array.from_array` in :func:`iris._lazy_data.as_lazy_data`, to avoid
+      the dask 'test access'.  This makes loading of netcdf files with a large number of variables significantly faster.
+      (:pull:`4135`) [``pre-v3.1.0``]
```

Comment:
```suggestion
      (:pull:`4135`)
```
Making this  as a suggestion (as raised by @bjlittle ), then hopefully we can easily add it to this PR